<ui version="4.0" >
 <class>WordList</class>
 <widget class="QDialog" name="WordList" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Bekannte Wörter</string>
  </property>
  <property name="windowIcon" >
   <iconset>:/images/tray.png</iconset>
  </property>
  <widget class="Line" name="line" >
   <property name="geometry" >
    <rect>
     <x>700</x>
     <y>0</y>
     <width>20</width>
     <height>551</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lbHelp" >
   <property name="geometry" >
    <rect>
     <x>720</x>
     <y>20</y>
     <width>270</width>
     <height>510</height>
    </rect>
   </property>
   <property name="text" >
    <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Hilfe:&lt;/p>
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Das ist die Hauptansicht für alle Aktionen&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">rund um das Spachmodell.&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Hier kannst du &lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;"> * Einzelne Wörter trainieren&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">    &lt;span style=" font-style:italic;">Für das allgemeine Training gibt es im &lt;/span>&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-style:italic;">    Hauptmenü einen extra Punkt &lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-style:italic;">    &lt;span style=" font-weight:600;">"Trainieren"&lt;/span>&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;"> * Wörter hinzufügen&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;"> * Wörter löschen&lt;/p>
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Um einzelne Wörter zu trainieren, markiere &lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">das Wort in der Tabelle links und ziehe es&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">in das Feld oben in der Mitte.&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Alternativ kannst du es auch im linken&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Auswahlfeld selektieren und den Knopf&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">"Zum Training hinzufügen" &lt;/span>drücken.&lt;/p>
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Der Knopf &lt;span style=" font-weight:600;">"Trainingsprogramm &lt;/span>&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-weight:600;">vorschlagen" &lt;span style=" font-weight:400;">selektiert automatisch &lt;/span>&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">zehn Wörter die noch&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Verbesserungspotential besitzen&lt;/p>
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Der Knopf "&lt;span style=" font-weight:600;">Neues Wort hinzufügen&lt;/span>"&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">öffnet einen neuen Dialog der Ihnen&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">helfen wird ein neues Wort hinzuzufügen.&lt;/p>
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">"&lt;span style=" font-weight:600;">Wort löschen&lt;/span>" &lt;span style=" text-decoration: underline;">löscht&lt;/span> das selektierte&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Wort &lt;span style=" text-decoration: underline;">unwiederbringlich&lt;/span>.&lt;/p>&lt;/body>&lt;/html></string>
   </property>
  </widget>
  <widget class="Line" name="line_3" >
   <property name="geometry" >
    <rect>
     <x>480</x>
     <y>480</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pbAddWord_2" >
   <property name="geometry" >
    <rect>
     <x>470</x>
     <y>440</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>Wort &amp;löschen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbAddWord" >
   <property name="geometry" >
    <rect>
     <x>470</x>
     <y>400</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>&amp;Neues Wort hinzufügen</string>
   </property>
  </widget>
  <widget class="Line" name="line_2" >
   <property name="geometry" >
    <rect>
     <x>480</x>
     <y>380</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pbBack" >
   <property name="geometry" >
    <rect>
     <x>470</x>
     <y>500</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text" >
    <string>&amp;Zurück</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbAddToTraining" >
   <property name="geometry" >
    <rect>
     <x>470</x>
     <y>260</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>Zum Training &amp;hinzufügen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSuggestTrain" >
   <property name="geometry" >
    <rect>
     <x>470</x>
     <y>340</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>Trainingsprogramm &amp;vorschlagen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbDeleteTrainingWord" >
   <property name="geometry" >
    <rect>
     <x>470</x>
     <y>300</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>Wort aus Liste &amp;löschen</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>480</x>
     <y>12</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text" >
    <string>Zum Trainieren ausgewählt:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbSearch_2" >
   <property name="geometry" >
    <rect>
     <x>17</x>
     <y>12</y>
     <width>134</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text" >
    <string>Gesamtes Vokabular:</string>
   </property>
   <property name="buddy" >
    <cstring>leSearch</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="lbSearch" >
   <property name="geometry" >
    <rect>
     <x>250</x>
     <y>13</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text" >
    <string>&amp;Suchen:</string>
   </property>
   <property name="buddy" >
    <cstring>leSearch</cstring>
   </property>
  </widget>
  <widget class="QPushButton" name="pbClearSearch" >
   <property name="geometry" >
    <rect>
     <x>430</x>
     <y>10</y>
     <width>16</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>&amp;x</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leSearch" >
   <property name="geometry" >
    <rect>
     <x>310</x>
     <y>10</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>leSearch</tabstop>
  <tabstop>pbClearSearch</tabstop>
  <tabstop>pbAddToTraining</tabstop>
  <tabstop>pbDeleteTrainingWord</tabstop>
  <tabstop>pbSuggestTrain</tabstop>
  <tabstop>pbAddWord</tabstop>
  <tabstop>pbAddWord_2</tabstop>
  <tabstop>pbBack</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
